scripts/neutron-band-structure.py

```diff
@@ -0,0 +1,209 @@
+#! /usr/bin/env python3
+import argparse
+import os
+import pathlib
+from typing import List, Tuple
+
+import euphonic
+from euphonic import ureg
+import euphonic.plot
+import matplotlib.pyplot as plt
+import numpy as np
+import seekpath
+
+
+def get_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser()
+    parser.add_argument('file', type=str,
+                        help=('File with force constants data. Supported '
+                              'formats: .yaml (Phonopy); .castep_bin, .check '
+                              '(Castep); .json (Euphonic)'))
+    parser.add_argument('--ebins', type=int, default=200)
+    parser.add_argument('--seekpath-labels', action='store_true',
+                        dest='seekpath_labels',
+                        help='Use the exact labels reported by Seekpath when '
+                             'constructing the band structure path. Otherwise,'
+                             ' use the labels detected by Euphonic.')
+    parser.add_argument('--q-distance', type=float, default=0.025,
+                        dest='q_distance',
+                        help='Target distance between q-point samples')
```

Comment:
Specify distance is in 1/angstrom

Reply:
Have allowed user to set q units; notation in help should look consistent with this option.